npm/pkg/dataplane/dataplane_windows.go

```diff
@@ -65,6 +77,14 @@ func (dp *DataPlane) initializeDataPlane() error {
 	return nil
 }
 
+func marshalFilterMap(filtermap map[string]uint16) ([]byte, error) {
+	filter, err := json.Marshal(filtermap)
+	if err != nil {
+		return nil, npmerrors.SimpleErrorWrapper("failed to marshal endpoint filter map", err)
```

Comment:
`npmerrors.SimpleError` doesn't look like it should exist since it's just an `error`--it doesn't add anything. Use the following from `github.com/pkg/errors` (you will probably need to add that import--the suggestion is not enough)

```suggestion
		return nil, errors.Wrap(err, "failed to marshal endpoint filter map")
```

Reply:
changed it to errors.Wrap, thank you for the suggestion